gemrb/core/Scriptable/Actor.cpp

```diff
@@ -11414,4 +11420,38 @@ void Actor::PlayArmorSound() const
 		PathTries = 0;
 	}
 
+	void Actor::BumpAway(Point farthest) {
+		if (!BumpBackTimer) {
+			OldPos = Pos;
+		}
+		BumpBackTimer = 10;
+		SetPosition(farthest, 0);
+	}
+
+	void Actor::BumpBack() {
+		assert(!BumpBackTimer);
+		SetPosition(OldPos, 0, 0, 0, false);
+	}
+
+	void Actor::DecreaseBumpBackTimer() {
+		if (!BumpBackTimer) {
+			return;
+		}
+		BumpBackTimer--;
+		if (!BumpBackTimer) {
+			BumpBack();
+		}
+	}
+
+bool Actor::WillBump() const {
+	return _WillBump;
+}
+
+void Actor::SetWillBump(bool willBump) {
+	_WillBump = willBump;
 }
```

Comment:
It looks like these two functions are misindented. Also, please rename the variable, as we clearly don't use leading underscores.